.gitignore

```diff
@@ -1 +1,7 @@
 _output/
+/shp
+/shp-*
+/bin/*
+
+# VSCode specific
+.vscode
```

Comment:
This should not be in the repo .gitignore, it should be in your global .gitignore.

Reply:
I just copied the bare minimal from [tektoncd/cli](https://github.com/tektoncd/cli/blob/main/.gitignore) but if you still want then I can revert this

Reply:
If everyone adds the directory for their development environment to every repository that they work on, it would become messy quite quickly. 

https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/

Reply:
addressed :+1: